backend/src/test/java/gov/cdc/usds/simplereport/test_util/SliceTestConfiguration.java

```diff
@@ -4,17 +4,17 @@
 import gov.cdc.usds.simplereport.config.AuditingConfig;
 import gov.cdc.usds.simplereport.config.AuthorizationProperties;
 import gov.cdc.usds.simplereport.config.InitialSetupProperties;
-import gov.cdc.usds.simplereport.config.authorization.AuthorizationServiceConfig;
-import gov.cdc.usds.simplereport.config.authorization.DemoUserIdentitySupplier;
+import gov.cdc.usds.simplereport.config.authorization.DemoAuthenticationConfiguration.DemoUserIdentitySupplier;
```

Comment:
This is the sneaky rename that makes you not notice that this. class is using that `DemoUserIdentitySupplier` constructor.